AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/ModelSyncedEventEmitter.swift

```diff
@@ -64,6 +64,10 @@ final class ModelSyncedEventEmitter {
             })
     }
 
+    deinit {
```

Comment:
As above, is this necessary, since Combine sinks are automatically cancelled upon release?